### PR TITLE
clean: Bump Scala to 2.13

### PR DESCRIPTION
**Pull request by lolgab**
- Update CircleCI build
- Bump sbt to 1.3.2
- Use codacy-orbs
- Use codacy-plugins-test orb

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,182 +1,61 @@
...
-# Re-usable blocks to reduce boilerplate in job definitions.
-references:
-  host_environment_defaults: &host_environment_defaults
-    # Customize the JVM maximum heap limit
-    JAVA_OPTS: -Xmx3200m
-  docker_environment_defaults: &docker_environment_defaults
-    # Customize the JVM to read correct memory values
-    JAVA_OPTS: '-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -XX:MaxRAMFraction=1'
+orbs:
+  codacy: codacy/base@0.3.14
```
Comment (ljmf00) [inline .circleci/config.yml]: Bump this

File: src/main/resources/docs/tests/example.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        blocks = [''.join(sorted(block, key=lambda s: s.lower())) for block in blocks]
+        # And the result is written back to README.md
+        sorted_file.write(''.join(blocks))
+
+    # Then we call the sorting method
+    sort_blocks()
+
+
+if __name__ == "__main__":
+    main()
```
Comment (ljmf00) [inline src/main/resources/docs/tests/example.py]: final line for consistency

File: .circleci/config.yml (hunk 1 above)
Comment (lolgab) [reply .circleci/config.yml]: Done ✅

File: src/main/resources/docs/tests/example.py (hunk 2 above)
Comment (lolgab) [reply src/main/resources/docs/tests/example.py]: Done ✅

**Pull request merged**